docker-compose-letsencrypt.yml

```diff
@@ -0,0 +1,206 @@
+    volumes:
+      - ./configs/frontend/:/etc/strelka/:ro
+      - logs:/var/log/strelka/
+    container_name: sublime_strelka_frontend_1
+    depends_on:
+      - sublime_strelka_coordinator
+  sublime_strelka_backend:
+    image: sublimesec/strelka-backend:0.3
+    restart: unless-stopped
+    command: strelka-backend
+    shm_size: 512mb
+    networks:
+      - net
+    volumes:
+      - ./configs/backend/:/etc/strelka/:ro
+    container_name: sublime_strelka_backend_1
+    depends_on:
+      - sublime_strelka_coordinator
+  sublime_strelka_manager:
+    image: sublimesec/strelka-manager:0.3
+    restart: unless-stopped
+    command: strelka-manager
+    container_name: sublime_strelka_manager_1
+    networks:
+      - net
+    volumes:
+      - ./configs/manager/:/etc/strelka/:ro
+    depends_on:
+      - sublime_strelka_coordinator
+  sublime_strelka_coordinator:
+    image: redis:alpine
+    restart: unless-stopped
+    command: redis-server --save "" --appendonly no
+    container_name: sublime_strelka_coordinator_1
+    networks:
+      - net
+  sublime_screenshot_service:
+    image: sublimesec/render-email-html:0.1
+    restart: unless-stopped
+    environment:
+      - S3_ENDPOINT=http://sublimes3:8110
+      - SCREENSHOT_BUCKET=email-screenshots
+      - AWS_REGION=us-east-1
+      - DISABLE_DD=true
+    container_name: sublime_screenshot_service
+    env_file: sublime.env
+    networks:
+      - net
+    depends_on:
+      - sublime_create_buckets
+  # Keep this name as sublimes3 because underscores don't play nice with certain endpoint validation
+  sublimes3:
+    container_name: sublimes3
+    image: minio/minio
+    restart: unless-stopped
+    networks:
+      - net
+    volumes:
+      - s3_data:/data
+    env_file: sublime.env
+    entrypoint: >
+      /bin/sh -c "
+      export MINIO_ROOT_USER=$$AWS_ACCESS_KEY_ID;
+      export MINIO_ROOT_PASSWORD=$$AWS_SECRET_ACCESS_KEY;
+      minio server --address 0.0.0.0:8110 --console-address 0.0.0.0:8111  /data;
+      "
+  sublime_create_buckets:
+    image: minio/mc
+    depends_on:
+      - sublimes3
+    networks:
+      - net
+    env_file: sublime.env
+    entrypoint: >
+      /bin/sh -c "
+      sleep 15;
+      /usr/bin/mc config host add myminio http://sublimes3:8110 $$AWS_ACCESS_KEY_ID $$AWS_SECRET_ACCESS_KEY;
+      /usr/bin/mc mb myminio/email-screenshots;
+      /usr/bin/mc ls myminio;
+      exit 0;
+      "
+  sublime_hydra:
+    image: sublimesec/hydra-cpu:0.1
+    restart: unless-stopped
+    networks:
+      - net
+    container_name: sublime_hydra
+
+  sublime_nginx_letsencrypt:
+    image: sublimesec/nginx-letsencrypt
+    build: ./nginx-letsencrypt/
+    restart: unless-stopped
+    ports:
+     - "80:80"
+     - "443:443"
+    depends_on:
+     - sublime_mantis
+     - sublime_dashboard
+    container_name: sublime_nginx_letsencrypt
+    env_file:
+     - certbot.env
+    volumes:
+     - /etc/letsencrypt/:/etc/letsencrypt/
+    networks:
+      - net
```

Comment:
this is the only new part compared to docker-compose.yml

Reply:
I think [Docker Compose Profiles](https://docs.docker.com/compose/profiles/) would be helpful here.